<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolLineDialog</class>
 <widget class="QDialog" name="ToolLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Tool</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="AttribLineWidget" name="line_widget" native="true">
       <property name="toolTip">
        <string>Double clic to start tool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="msg_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="start_btn">
         <property name="toolTip">
          <string>Start tool</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>ressources/icon_point.png</normaloff>ressources/icon_point.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removelast_btn">
         <property name="text">
          <string>Remove Last</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="toolTip">
          <string>Close dialog</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttribLineWidget</class>
   <extends>QWidget</extends>
   <header>attrib_line_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
